packages/@aws-cdk/assertions/test/private/section.test.ts

```diff
@@ -18,7 +18,7 @@ describe('section', () => {
       // THEN
       expect(result.match).toEqual(true);
       const success = result as MatchSuccess;
-      expect(success.matches.length).toEqual(2);
+      expect(Object.keys(success.matches).length).toEqual(2);
```

Comment:
can we update this test so we actually check the returned matches (rather than just the length)?

(sorry this was missed previously)